tasks/install_pxb_innovation_lts.yml

```diff
@@ -0,0 +1,25 @@
+# This task install Percona XtraBackup 8.1 on CentOS and Debian/Ubuntu
```

Comment:
Change version here + when Mohit is back from vacation - coordinate with him (we can commit like this and update it later.)